pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -812,6 +820,15 @@ public void checkMessageDeduplicationInfo() {
         });
     }
 
+    public void checkInactiveSubscriptions() {
+        topics.forEach((n, t) -> {
+            Topic topic = t.getNow(null);
```

Comment:
`getNow()` throws exception if the topic loading failed and the future wasn't cleaned up yet

Reply:
Do I have to fix 't.getNow(null);' of other parts? (e.g. 
https://github.com/apache/incubator-pulsar/pull/1352/files/320a0f71cfb5c8c8fe67430806281417b6982120#diff-bb90edad9f69a5d791cb43b04d9a4c82R816)

Reply:
Yes please, the correct way should be : 

```java
Topic topic = t.isCompletedExceptionally() ? null : t.getNow(null);
```

Reply:
I see, thank you.